### PR TITLE
[17.0][MIG] hr_job_category

### PR DESCRIPTION
**Pull request by imlopes**
ref: C2C-2110

### CODE REVIEW

File: hr_job_category/tests/test_hr_job_categories.py [hunk 1]
```diff
@@ -108,15 +108,15 @@ def test_remove_tags_from_previous_job(self):
...
+        self.contract_id.invalidate_recordset()
 
         # We have two tags (from job and the manual added one)
         self.assertEqual(len(self.employee_id_1.category_ids.ids), 2)
 
         # We change the contract of the employe
         # We should now have the tag
         self.contract_id.write({"job_id": self.job_2_id.id})
-        self.contract_id.flush()
+        self.contract_id.invalidate_recordset()
```
Comment (ivantodorovich) [inline hr_job_category/tests/test_hr_job_categories.py]: `flush_recordset` perhaps? although it seems useless, we might be able to remove this line

File: hr_job_category/tests/test_hr_job_categories.py [hunk 2]
```diff
@@ -61,7 +61,7 @@ def test_write_computes_with_normal_args(self):
 
         # Check if job2 categories are written to the employee
         self.contract_id.write({"job_id": self.job_2_id.id})
-        self.contract_id.flush()
+        self.contract_id.invalidate_recordset()
```
Comment (ivantodorovich) [inline hr_job_category/tests/test_hr_job_categories.py]: `flush_recordset` perhaps? although it seems useless, we might be able to remove this line

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Could you please rename the commit to `[MIG] Migration to 17.0` ? 🙏🏻

File: hr_job_category/tests/test_hr_job_categories.py (hunk 1 above)
Comment (imlopes) [reply hr_job_category/tests/test_hr_job_categories.py]: @ivantodorovich do you mean remove all `invalidate_recordset` ?

Comment (ivantodorovich) [reply hr_job_category/tests/test_hr_job_categories.py]: No, just the lines that called `flush` before

File: hr_job_category/__manifest__.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "HR Job Employee Categories",
-    "version": "15.0.1.0.1",
+    "version": "17.0.1.0.1",
```
Comment (yankinmax) [inline hr_job_category/__manifest__.py:6]: we shall specify version: `"version": "17.0.1.0.0"`

**Review by yankinmax — APPROVED** [review]

File: hr_job_category/__manifest__.py (hunk 3 above)
Comment (ivantodorovich) [reply hr_job_category/__manifest__.py:6]: indeed 👍🏻

**Comment by imlopes** [comment]
@pedrobaeza can you check this PR too, please ?

**Comment by pedrobaeza** [comment]
/ocabot migration hr_job_category

Sorry, I don't use this module.

**Comment by simahawk** [comment]
> Could you please rename the commit to `[MIG] Migration to 17.0` ? 🙏🏻

it should contain the module name in fact :)

File: hr_job_category/readme/INSTALL.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+To install this module, you need to:
+
```
Comment (simahawk) [inline hr_job_category/readme/INSTALL.md]: IMO we can trash this file (which BTW refers to v11)

File: hr_job_category/tests/test_hr_job_categories.py [hunk 5]
```diff
@@ -0,0 +1,134 @@
+# Copyright 2014 Savoir-faire Linux
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestHrJobCategories(common.TransactionCase):
```
Comment (simahawk) [inline hr_job_category/tests/test_hr_job_categories.py]: please use BaseCommon

File: hr_job_category/tests/test_hr_job_categories.py [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+    def setUp(self):
```
Comment (simahawk) [inline hr_job_category/tests/test_hr_job_categories.py]: Please rework this w/ setUpClass

**Review by simahawk — CHANGES_REQUESTED** [review]
If you apply the changes in separate commits we can backport :wink:

**Comment by imlopes** [comment]
Hello @simahawk 
Can you re-check this PR, please ?

**Review by simahawk — APPROVED** [review]

**Pull request merged**